Docs edits (#706)
omry/omegaconf: docs/source/usage.rst

    host: localhost
    <BLANKLINE>

> OmegaConf objects constructed from Structured classes offer runtime type safety:

Reviewer: ```suggestion
OmegaConf objects constructed from Structured classes provide runtime type safety:
```

Struct flag
^^^^^^^^^^^
By default, OmegaConf dictionaries allow read and write access to unknown fields.

> If a field does not exist, writing it will create the field, and attempting to
> access the field before creation will raise an exception (either ``ConfigKeyError``
> or ``ConfigAttributeError``, depending on the mode of access).
> It's sometime useful to change this behavior.  Using ``OmegaConf.set_struct``,
> it is possible to prevent writing to fields that do not exist:

Reviewer: ```suggestion
If a field does not exist, writing it will create the field, and attempting to
access the field before creation will raise an exception (either ``ConfigKeyError``
or ``ConfigAttributeError``, depending on the mode of access).
It's sometime useful to change this behavior.  Using ``OmegaConf.set_struct``,
it is possible to prevent the creation of fields that do not exist:
```

OmegaConf.to_object
^^^^^^^^^^^^^^^^^^^^^^
The ``OmegaConf.to_object`` method recursively converts ``DictConfig`` and ``ListConfig`` objects

> into native Python dicts and lists, with the exception that Structured Config objects are

Reviewer: I prefer plain dicts and lists:
```suggestion
into plain Python dicts and lists, with the exception that Structured Config objects are
```

> converted into instances of the backing dataclass or attr class.  All OmegaConf
> interpolations are resolved before conversion to Python containers.

Reviewer: ```suggestion
converted into instances of the backing dataclass or attr class.  Interpolations in the config are always resolved by ``OmegaConf.to_object``.
```
